src/App.test.js

```diff
@@ -1,8 +1,54 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import userEvent from '@testing-library/user-event';
+import App from './app';
 
-test('renders learn react link', () => {
+window.scroll = jest.fn();
```

Comment:
Do we need this line? unless I'm missing something, I don't think we're using it anywhere.

Reply:
Without this line the test is throwing an error: 
`Error: Not implemented: window.scroll`

I understand that this is due to jest using jsdom which simulates some browser functionality but not all. The error points to our handleNavigate function which calls window.scroll. A very similar problem is described here (window.scrollTo): https://stackoverflow.com/questions/57311971/error-not-implemented-window-scrollto-how-do-we-remove-this-error-from-jest-t

I was not able to solve it by adding --env=jsdom to the test script in package.json so I choose this mocking solution. Perhaps we should also add afterAll/afterEach logic to clear the mocks if we keep the jest.fn() solution?